packages/members-api/lib/repositories/member.js

```diff
@@ -174,7 +174,7 @@ module.exports = class MemberRepository {
 
         if (context.internal) {
             source = 'system';
-        } else if (context.user) {
```

Comment:
Do we want this to be admin or would it not be more useful to have this be `importer` ?

Reply:
If I can choose, then I'd probably prefer to have its own source e. g. `importer`. I wasn't sure if we could add a new source.

Reply:
Thought about this a bit more and I think `importer` would be the right choice. That way we can include members that have been manually added by the site owner in our calculation if we'd want to and solely exclude imported ones and their MRR

Reply:
I've added a new `importer` source now